src/OneDrive.js

```diff
@@ -423,7 +423,11 @@ export class OneDrive {
   /**
    */
   getWorkbook(driveItem) {
-    return new Workbook(this, `/drives/${driveItem.parentReference.driveId}/items/${driveItem.id}/workbook`, this.log);
+    return this.getWorkbookFromPath(`/drives/${driveItem.parentReference.driveId}/items/${driveItem.id}/workbook`);
+  }
+
+  getWorkbookFromPath(path) {
```

Comment:
what is the use-case of that method?

Reply:
Githib actions implementation for post-submit processing. Franklin passes us the raw path in the event payload and we'd like to create a workbook object out of that https://github.com/hegupta-adbe/form-data-processor/blob/helix-onedrive/index.js#L202

Reply:
I'd rather use the [OneDrive.getDriveItem](https://github.com/adobe/helix-onedrive-support/blob/eff748fa9c1daeacd9c97594241a0e607da0ecae/src/OneDrive.d.ts#L150) API followed by [OneDrive.getWorkbook](https://github.com/adobe/helix-onedrive-support/blob/eff748fa9c1daeacd9c97594241a0e607da0ecae/src/OneDrive.d.ts#L190). Your code change would return a `Workbook` instance with an opaque path that has neither been validated nor tested for existence.

Reply:
or at least: https://github.com/adobe/helix-onedrive-support/blob/eff748fa9c1daeacd9c97594241a0e607da0ecae/src/OneDrive.d.ts#L112C10-L112C26